python/lsst/ts/fbs/utils/maintel/basis_functions.py

```diff
@@ -243,32 +247,26 @@ def get_basis_functions_field_survey(
         # replace airmass basis function)
         basis_functions.AltAzShadowMaskBasisFunction(
             nside=nside,
-            min_alt=22,
-            max_alt=83,
+            min_alt=40.0,
```

Comment:
Sorry, I think I wasn't clear about this previous comment. I think it's okay to leave the min_alt=22.0 down here in the base class of the basis_functions as the "general" set of constraints. But that we should add a second, more restrictive AltAzShadowMaskBasisFunction at the ts_config_ocs level that allows us to quickly apply updates based on current conditions (e.g. the min_alt restriction set by M1M3, which has since moved from 40.0 to 30.0).

Reply:
Maybe we should expose this values to the user in the method argument?

Reply:
Now passing in min_alt and max_alt from the ts_config_ocs level.